src/ServicePulse.Host/app/js/dashboard/spMoment.ts

```diff
@@ -0,0 +1,27 @@
+import moment from 'moment';
+
+export default class EventLogItemComponent extends HTMLElement {
+    date: moment.Moment | null = null;
+
+    constructor() {
+        super();
+
+        // Wait for slots to be populated
+        let observer = new MutationObserver((changes, observer) => {
```

Comment:
This web component shows how sub-web components would have to be loaded and displayed. This observer mechanism is again only required because of a lack of shadow dom.